socorro/external/boto/connection_context.py

```diff
@@ -254,29 +233,23 @@ def _convert_list_to_string(self, a_list):
     def _convert_string_to_list(self, a_string):
         return json.loads(a_string)
 
-    def commit(self):
-        """boto doesn't support transactions so this silently
-        does nothing"""
-
-    def rollback(self):
-        """boto doesn't support transactions so this silently
-        does nothing"""
-
     @contextlib.contextmanager
     def __call__(self):
         yield self
 
-    def in_transaction(self, dummy):
-        """boto doesn't support transactions, so it is never in
-        a transaction."""
-        return False
-
     def force_reconnect(self):
-        try:
-            del self.connection
-        except AttributeError:
-            # already deleted, ignorable
-            pass
```

Comment:
Boto does its stuff via HTTPS, so I don't think there's a persistent connection to reconnect. Given that, I just nixed this code.